### PR TITLE
CTOR-922 : Refresh network-aruba-orchestrator-restapi

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh network-aruba-orchestrator-restapi

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-aruba-orchestrator-restapi.md [hunk 1]
```diff
@@ -62,70 +69,177 @@ pour en savoir plus sur la découverte automatique d'hôtes.
...
-    --port='443' \
-    --proto='https' \
-    --api-token='mytoken' \
-    --verbose
+	--plugin=network::aruba::orchestrator::restapi::plugin \
+	--mode=appliances \
+	--hostname='10.0.0.1' \
+	--port='443' \
+	--proto='https' \
+	--access-token=''  \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-aruba-orchestrator-restapi.md:234]: --access-token or --api-token? mandatory value?

File: pp/integrations/plugin-packs/procedures/network-aruba-orchestrator-restapi.md [hunk 2]
```diff
@@ -5,54 +5,61 @@ title: Aruba Orchestrator Rest API
...
 </TabItem>
 </Tabs>
 
+### Discovery rules
+
+#### Host discovery
+
+| Rule name                    | Description                            |
+|:-----------------------------|:---------------------------------------|
+| Aruba Orchestrator Appliance | Discover Aruba Orchestrator Appliances |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-aruba-orchestrator-restapi.md:37]:
```suggestion
| Aruba Orchestrator Appliance | Discover Aruba Orchestrator appliances |
```

File: pp/integrations/plugin-packs/procedures/network-aruba-orchestrator-restapi.md [hunk 3]
```diff
@@ -61,70 +68,177 @@ More information about discovering hosts automatically is available on the [dedi
...
-    --port='443' \
-    --proto='https' \
-    --api-token='mytoken' \
-    --verbose
+	--plugin=network::aruba::orchestrator::restapi::plugin \
+	--mode=appliances \
+	--hostname='10.0.0.1' \
+	--port='443' \
+	--proto='https' \
+	--access-token=''  \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-aruba-orchestrator-restapi.md:233]: same as FR

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-aruba-orchestrator-restapi.md (hunk 1 above)
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-aruba-orchestrator-restapi.md:234]:
```suggestion
	--access-token='XXXX'  \
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**